modules/core/src/core/attribute.js

```diff
@@ -122,11 +122,11 @@ export default class Attribute {
     this.divisor = divisor;
 
     if (isInstanced !== undefined) {
-      log.deprecated('Attribute.isInstanced');
+      log.deprecated('Attribute.isInstanced')();
       this.divisor = isInstanced ? 1 : 0;
     }
     if (instanced !== undefined) {
-      log.deprecated('Attribute.instanced');
+      log.deprecated('Attribute.instanced')();
```

Comment:
you need a linter rule for this!

Reply:
> you need a linter rule for this!

Yes, that is a good idea! Do you know if it possible to define custom linter rules for our own APIs? Any link to how to do it?

Reply:
https://github.com/uber/luma.gl/commit/ef34ca48db2fceb42714fe8891cf8951440f1041